### PR TITLE
R11runner 05 day5

### PR DESCRIPTION
**Pull request by ghost**
Today only a small refactoring.
But the tests are still not really readable.

### CODE REVIEW

File: 2_adventures/001/r11runner/05_day5/dictionary_entry_test.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+require 'minitest/autorun'
+require_relative './dictionary_entry.rb'
```
Comment (kotp) [inline 2_adventures/001/r11runner/05_day5/dictionary_entry_test.rb:2]: require_relative requires no "this folder" dot.

**Comment by kotp** [comment]
lgtm

**Comment by patmaddox** [comment]
Go you!

**Comment by ghost** [comment]
@kotp Thanks for the comment about require :)

**Pull request merged**